internal/provider/resource_schema.go

```diff
@@ -834,11 +834,13 @@ func loadSchema(ctx context.Context, d *schema.ResourceData, c *SchemaRegistryRe
 func readSchemaRegistryConfigAndSetAttributes(ctx context.Context, d *schema.ResourceData, c *SchemaRegistryRestClient, subjectName string, schemaIdentifier string) (*sr.Schema, error) {
 	isLatestSchemaBool := isLatestSchema(schemaIdentifier)
 	srSchema, exists, err := loadSchema(ctx, d, c, subjectName, schemaIdentifier)
-	if err != nil {
+	if !exists && err != nil {
```

Comment:
That's an excellent start! However, it appears I may be overlooking something, so I have a quick question:

Let's examine two scenarios for [the code block](https://github.com/confluentinc/terraform-provider-confluent/blob/master/internal/provider/resource_schema.go#L809-L812) where
```
schemaIdentifier, err = loadIdForLatestSchema(ctx, d, c, subjectName)
```
fails with:

1. A 404 HTTP response status code and err != nil.
2. A 403 HTTP response status code and err != nil.
3. A 5xx HTTP response status code and err != nil.

It seems like in all three scenarios, `!exists && err != nil` will be true, and
```
return nil, fmt.Errorf("error reading Schema %q: %s", d.Id(), createDescriptiveError(err))
```
will be returned. However, we'd want to achieve `d.SetId("")` for a 404 and return
```
return nil, fmt.Errorf("error reading Schema %q: %s", d.Id(), createDescriptiveError(err))
```
for 403 and 5xx HTTP response status codes.